.github/workflows/deploy.yml

```diff
@@ -84,7 +98,11 @@ jobs:
         working-directory: site/gatsby-site
         run: |
           set -e
-          OUTPUT=$(bash -c "netlify deploy --json --alias=pr-${{ github.event.pull_request.number }}" | tr '\n' ' ')
+          if [[ -z "${{ inputs.alias }}" ]]; then
```

Comment:
This logic is worth inline comments of the expected outputs. If I read this correctly...

"If the netlify alias is not set, target the "production" environment of the site. Otherwise, deploy to the given alias."